docs/source/roles/researcher/troubleshooting.md

```diff
@@ -36,6 +36,18 @@ Either log into that workspace and close Firefox, or follow the instructions [he
 Your profile is likely stored under **~/snap/firefox/common/.mozilla/firefox**.
 :::
 
+## {{zzz}} Idle screen lock
+
+By default, the Xfce desktop will enter a screensaver and lock the screen after idling for five minutes.
+This will require entering your password to unlock and is in addition to authenticating with Guacamole and the lock screen on your own computer.
+
+You can disable the Xfce lock screen if you find it is unnecessary and slows your work.
+
+First, open Screensaver Preferences either by nagivating to **{guilabel}`Applications --> Settings --> Screensaver Preferences`** or by running `xfce4-screensaver-preferences` on the command line.
```

Comment:
Should this whole section be: (a) a list and (b) wrapped in a dropdown?

Reply:
Yes, I think both makes it more consistent and looks better.